designs/developer/index.md

```diff
@@ -5,95 +5,99 @@ include_designs_by_version: false
 
 # Developer
 
+## Overview
 The Developer Perspective, added to the 4.2 OpenShift Console, provides developers with an optimized experience with the features and workflows they’re most likely to need to be productive. Developers can focus on higher level abstractions like their application and components, and then drill down deeper to get to the OpenShift and Kubernetes resources that make up their application.
 
 We strive for simplicity, while still allowing for users to achieve more advanced flows.  We focus on efficiency for those workflows that are more frequently utilized.
 
 Our design documentation covers the entire developer experience in the OpenShift Console.  The developer experience is enhanced as operators are installed, which is also documented below.
 
-## Personas and Archetypes
--- coming soon --
+## Main Use Cases 
+The main use cases of the Developer Perspective include: 
+- Utilizing the topology view to view resources 
+- Adding resources to your topology 
+- Monitoring your resources
+- Modifying your resources 
+- Enchancing the Developer Perspective with operators 
 
-## 4.5 Developer Experience
-The following section documents the 4.5 OpenShift Console Developer experience.
+## Personas
+Here we could include brief information about the sorts of personas who are the main users of the developer perspective. 
 
-### Masthead
-The masthead is common to both the Developer and Administrator Perspectives. Listed below are features that are accessible through the masthead.
-- [Command Line Interface \(CLI\)](https://openshift.github.io/openshift-origin-design/designs/developer/cli/)
+Some of the main users who utilize the Developer Perspective include: 
+- Developers  
+- User type 
+- User type
 
-### Primary Navigation
-The primary navigation in the Developer Perspective has three main groupings: task based, object based, and an object based section of resources which have been added to the navigation by the user.
+# Releases 
 
-Task based sections:
-- [+Add](https://openshift.github.io/openshift-origin-design/designs/developer/add-45/)
-- [Topology](https://openshift.github.io/openshift-origin-design/designs/developer/topology-44/)
-- [Monitoring](https://openshift.github.io/openshift-origin-design/designs/developer/monitoring-44/)
-- [Search](https://https://openshift.github.io/openshift-origin-design/designs/developer/search-45/)
+4.x
 
-Object based sections:
-- [Builds](https://openshift.github.io/openshift-origin-design/designs/developer/builds/)
-- [Pipelines](https://openshift.github.io/openshift-origin-design/designs/developer/operator-pipeline-45/pipelines)
-- [Helm](https://openshift.github.io/openshift-origin-design/designs/developer/helm-45/)
-- [Project](https://openshift.github.io/openshift-origin-design/designs/developer/project-45/)
+- Link to design docs
```

Comment:
dumb question but why do we need all this "Link to design docs" text here?

Reply:
Those are placeholders now for the time being since we're updating the formatting.